dts/arm/st/h5/stm32h563.dtsi

```diff
@@ -9,5 +9,15 @@
 / {
 	soc {
 		compatible = "st,stm32h563", "st,stm32h5", "simple-bus";
+
+		sdmmc2: sdmmc@46008c00 {
```

Comment:
If  the sm32h562 device has 2 SDMMC instances, then SDMMC 1 & 2 are nodes of the stm32h562.dtsi

Reply:
I think there is only one in sm32h562, but I'll have a second look

Reply:
I confirm SDMM2 is only available on stm32h563.